a2_questionnaire_design_part_a

gzuo opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)

## What did you learn from the changes you have made?

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

## How were these changes tested?

## A reference to a related issue in your repository (if applicable)

## Checklist
- [X] I can confirm that my changes are working as intended

KasraVakiloroayaei commented:
A2: INCOMPLETE 
Please specify which sampling strategy would be most appropriate here (Simple random sample/ Starta/ Cluster etc..). 
I would also suggest to avoid open ended questions on a questionnaire, as it is hard to collect data of that nature.

gzuo commented:
> A2: INCOMPLETE Please specify which sampling strategy would be most appropriate here (Simple random sample/ Starta/ Cluster etc..). I would also suggest to avoid open ended questions on a questionnaire, as it is hard to collect data of that nature.

@Melokazz - Thank you for the comment and suggestions!  I've updated the assignment as following.  When you get a chance, would you please verify it?
 - Specified which sampling strategy would be most appropriate here - stratified random sampling.
 - Changed all open ended questions by providing options.

Thank you!
George
